mad-dog/package.json

```diff
@@ -4,8 +4,7 @@
   "description": "🐶 Audius Protocol System Level Test Suite",
   "main": "src/index.js",
   "scripts": {
-    "start": "node src/index.js --inspect",
```

Comment:
There's no libs dep here anyway, and I don't think anyone is calling this manually

cc @piazzatron

Reply:
interesting, i didnt realize we had removed these deps
but also, there is a service-commands dep in maddog, how does this work without specifying it here?

Reply:
it's linked in init-repos.sh. not sure what you mean? this was just a helper to run `npm run link-deps` which I didn't see us calling anywhere

Reply:
my b, couple diff points
1. i know its linked in init-repos, makes sense to remove this helper. i guess this helper predated that init-repos script, and i think at some point manually re-linking deps here was useful. anyway, all good 👍 
2. mad-dog does actually consume service-commands, so i'm surprised the service-commands dependency is not listed in package.json. obviously it doesn't matter since we only ever run maddog with libs linked

Reply:
ill add a subsequent PR w/ those deps. definitely not correct

Reply:
🙏 dope